Makefile

```diff
@@ -129,6 +139,7 @@ test: build
 	./vendor/bin/phpunit --coverage-clover clover.xml -c phpunit.xml
 
 clean:
+	l10n-clean
```

Comment:
just mind debugging this - are you sure that `make clean` will not call `l10-clean` command (instead of executign sub task?)

Reply:
D'Oh!.
You are right. ..